searx/plugins/__init__.py

```diff
@@ -54,7 +62,9 @@ def __iter__(self):
         for plugin in self.plugins:
             yield plugin
 
-    def register(self, *plugins):
+    def register(self, *plugins, external=False):
+        if external:
+            plugins = load_external_plugins(plugins)
```

Comment:
check of the mandatory names (`required_attrs`) is missed for external plugins

Reply:
Why? It happens later in this function.